--- packages/perl-modules/lib/BuildHelper.pm	2017-11-17 15:07:47.000000000 +0100
+++ packages/perl-modules/lib/BuildHelper_v2.pm	2017-12-04 19:55:15.258747201 +0100
@@ -430,7 +430,7 @@

     eval {
         local $SIG{ALRM} = sub { die "timeout on: $file\n" };
-        alarm(120); # single module should not take longer than 1 minute
+        alarm(1200); # single module should not take longer than 1 minute
         if( -f "Build.PL" ) {
             `$PERL Build.PL >> $LOG 2>&1 && $PERL ./Build >> $LOG 2>&1 && $PERL ./Build install >> $LOG 2>&1`;
             if($? != 0 ) { die("error: rc $?\n".`cat $LOG`."\n"); }
