--- defines.make	2023-05-22 09:36:30.000000000 +0100
+++ defines.make_v2	2023-06-09 09:12:40.627821729 +0100
@@ -88,14 +88,7 @@
 # PyPi Mirror Configuration
 # By default our internal Python mirror is used.
 # To use the official Python mirror, please export `USE_EXTERNAL_PIPENV_MIRROR=true`.
-EXTERNAL_PYPI_MIRROR := https://pypi.python.org/simple
-INTERNAL_PYPI_MIRROR :=  https://devpi.lan.tribe29.com/root/pypi
-
-ifeq (true,${USE_EXTERNAL_PIPENV_MIRROR})
-PIPENV_PYPI_MIRROR  := $(EXTERNAL_PYPI_MIRROR)
-else
-PIPENV_PYPI_MIRROR  := $(INTERNAL_PYPI_MIRROR)
-endif
+PIPENV_PYPI_MIRROR := https://pypi.python.org/simple
 
 print-%:
 	@echo '$($*)'
