--- Makefile	2023-05-12 00:02:21.000000000 +0100
+++ Makefile_v2	2023-06-09 09:15:20.787179676 +0100
@@ -296,6 +296,7 @@
 .INTERMEDIATE: .ran-webpack
 $(JAVASCRIPT_MINI): .ran-webpack
 $(THEME_CSS_FILES): .ran-webpack
+.ran-webpack: export NODE_OPTIONS := --max-old-space-size=2048
 .ran-webpack: node_modules/.bin/webpack webpack.config.js postcss.config.js $(JAVASCRIPT_SOURCES) $(SCSS_SOURCES)
 	WEBPACK_MODE=$(WEBPACK_MODE) ENTERPRISE=$(ENTERPRISE) MANAGED=$(MANAGED) PLUS=$(PLUS) node_modules/.bin/webpack --mode=$(WEBPACK_MODE:quick=development)
 	touch $(JAVASCRIPT_MINI) $(THEME_CSS_FILES)
