--- omd/packages/heirloom-mailx/heirloom-mailx_http.bzl 2023-05-12 01:02:21.000000000 +0200
+++ omd/packages/heirloom-mailx/heirloom-mailx_http.bzl_v2      2023-05-28 17:44:04.992958010 +0200
@@ -5,8 +5,7 @@
     http_archive(
         name = "heirloom-mailx",
         urls = [
-            "https://ftp.debian.org/debian/pool/main/h/heirloom-mailx/heirloom-mailx_" + HEIRLOOMMAILX_VERSION + ".orig.tar.gz",
-            "https://artifacts.lan.tribe29.com/repository/upstream-archives/heirloom-mailx_" + HEIRLOOMMAILX_VERSION + ".orig.tar.gz",
+            "http://archive.ubuntu.com/ubuntu/pool/universe/h/heirloom-mailx/heirloom-mailx_" + HEIRLOOMMAILX_VERSION + ".orig.tar.gz",
         ],
         sha256 = "015ba4209135867f37a0245d22235a392b8bbed956913286b887c2e2a9a421ad",
         build_file = "@omd_packages//packages/heirloom-mailx:BUILD.bazel",
