--- packages/nrpe/Makefile	2017-12-04 20:03:26.836924104 +0100
+++ packages/nrpe/Makefile_v2	2017-12-04 20:04:20.474543086 +0100
@@ -17,7 +17,7 @@
 	    echo "applying $$p..." ; \
 	    patch -p1 -b -d $(DIR) < $$p ; \
 	done
-	cd $(DIR) ; ./configure $(CONFIGUREOPTS) ; echo "#include \"../../libssl_compat.h\"" > temp.h ;
+	cd $(DIR) ; ./configure $(CONFIGUREOPTS) ; echo "#include \"../../libssl_compat.h\"" > temp.h ; cat include/dh.h >> temp.h ; mv temp.h include/dh.h
 	$(MAKE) -C $(DIR) all

 install:
