--- Pipfile	2022-05-26 01:02:30.000000000 +0100
+++ Pipfile_v2	2022-06-11 19:53:11.919789661 +0100
@@ -36,7 +36,6 @@
 types-redis = "==4.1.15"  # used for type checking
 types-protobuf = "==3.19.12"  # used for type checking
 types-setuptools = "==57.4.9"  # used for type checking
-playwright = "==1.19.0"  # used for in-browser testing
 pydantic-factories = "==1.2.4"  # used for pydantic model testing
 pymongo = "==4.0.1"  # used by mk_mongodb agent plugin
 pytest = "==7.0.1"  # used by various test/Makefile targets
