--- Pipfile	2023-05-12 00:02:21.000000000 +0100
+++ Pipfile_v2	2023-06-09 09:22:34.125378189 +0100
@@ -37,7 +37,6 @@
 types-setuptools = "==67.2.0.0"  # used for type checking
 types-pyOpenSSL = "==23.0.0.0"  # used for type checking
 types-waitress = "==2.1.4.4"  # used for type checking
-playwright = "==1.30.0"  # used for in-browser testing
 pydantic-factories = "==1.17.1"  # used for pydantic model testing;
 pymongo = "==4.3.3"  # used by mk_mongodb agent plugin
 pytest = "==7.2.1"  # used by various test/Makefile targets
