--- Pipfile     2023-05-12 01:02:21.000000000 +0200
+++ Pipfile_v2  2023-05-28 17:25:11.054329162 +0200
@@ -37,7 +37,6 @@
 types-setuptools = "==67.2.0.0"  # used for type checking
 types-pyOpenSSL = "==23.0.0.0"  # used for type checking
 types-waitress = "==2.1.4.4"  # used for type checking
-playwright = "==1.30.0"  # used for in-browser testing
 pydantic-factories = "==1.17.1"  # used for pydantic model testing;
 pymongo = "==4.3.3"  # used by mk_mongodb agent plugin
 pytest = "==7.2.1"  # used by various test/Makefile targets
@@ -100,7 +99,6 @@
 chardet = "==5.1.0"  # needed by requests
 urllib3 = "==1.26.13"  # needed by requests
 pyOpenSSL = "==19.1.0"  # needed by requests with extras = ["security"]
-pbr = "==5.11.0"  # needed by jira
 pyghmi = "==1.5.13"  # needed by base for IPMI
 requests = {version = "==2.28.1",extras = ["security"]}  # needed by DCD, connexion
 pykerberos = "==1.2.1"  # needed by check_bi_aggr
