--- omd/packages/protobuf/protobuf.make	2023-05-12 00:02:22.000000000 +0100
+++ omd/packages/protobuf/protobuf.make_v2	2023-06-09 09:21:09.395735669 +0100
@@ -65,7 +65,7 @@
 	    `: Let me know in case you got a cleaner approach. ` \
 	    cd src && \
 	    rm protoc && \
-	    echo -e '\nprotoc-static: $(protoc_OBJECTS) $(protoc_DEPENDENCIES) $(EXTRA_protoc_DEPENDENCIES)\n\tg++ -pthread -DHAVE_PTHREAD=1 -DHAVE_ZLIB=1 -Wall -Wno-sign-compare -static-libgcc -static-libstdc++ -s -o protoc google/protobuf/compiler/main.o -lpthread ./.libs/libprotoc.a ./.libs/libprotobuf.a' >> Makefile && \
+	    echo -e '\nprotoc-static: $(protoc_OBJECTS) $(protoc_DEPENDENCIES) $(EXTRA_protoc_DEPENDENCIES)\n\tg++ -pthread -DHAVE_PTHREAD=1 -DHAVE_ZLIB=1 -Wall -Wno-sign-compare -static-libgcc -static-libstdc++ -s -o protoc google/protobuf/compiler/main.o -lpthread ./.libs/libprotoc.a ./.libs/libprotobuf.a -latomic' >> Makefile && \
 	    make -j6 protoc-static && \
 	    file $(PROTOBUF_BUILD_DIR)/src/protoc | grep ELF >/dev/null && \
 	    ldd $(PROTOBUF_BUILD_DIR)/src/protoc | grep -v libstdc++ >/dev/null
