--- omd/packages/Python/Python.make	2023-05-12 00:02:21.000000000 +0100
+++ omd/packages/Python/Python.make_v2	2023-06-09 09:17:50.516565773 +0100
@@ -92,7 +92,8 @@
 	        --with-ensurepip=install \
 	        --with-openssl=$(PACKAGE_OPENSSL_DESTDIR) \
 	        $(PYTHON_ENABLE_OPTIMIZATIONS) \
-	        LDFLAGS="-Wl,--rpath,/omd/versions/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx/lib $(PACKAGE_OPENSSL_LDFLAGS)"
+	        CFLAGS="${CFLAGS} -fno-semantic-interposition" \
+	        LDFLAGS="-Wl,--rpath,/omd/versions/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx/lib -fno-semantic-interposition $(PACKAGE_OPENSSL_LDFLAGS)"
 	cd $(PYTHON_BUILD_DIR) ; \
 	    $(MAKE) -j2
 	$(TOUCH) $@
